templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
These templates already existed from previous work and we were not interested in re-working them as YAML. While I understand and mostly agree with your comment, I'm still not really interested in spending the effort when both are equally supported.